### PR TITLE
Header component made

### LINKED ISSUE #1: Header Component

### PR DESCRIPTION
**Pull request by JustinLung**

### CODE REVIEW

File: src/lib/components/Header.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	<img src="/assets/images/logo.svg" alt="Hogeschool van Amsterdam Logo" />
+	<span>Frontend <br /> Design & Development</span>
+</header>
+
+<style>
+	header {
+		display: flex;
+		justify-content: flex-start;
+		align-items: center;
+        color: var(--color-white);
```
Comment (boudewijnbout) [inline src/lib/components/Header.svelte]: Weird spacing?

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <script lang="ts">
-	import '../styles/stylecss';
```
Comment (boudewijnbout) [inline src/routes/+layout.svelte:2]: Typo

File: src/styles/style.css [hunk 3]
```diff
@@ -1,4 +1,7 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
```
Comment (boudewijnbout) [inline src/styles/style.css]: @import is a bad way of importing fonts (bad performance). Consider using a google font link in the +layout.svelte.

**Review by boudewijnbout — CHANGES_REQUESTED** [review]
Improvents could be made.

File: src/routes/+layout.svelte (hunk 2 above)
Comment (JustinLung) [reply src/routes/+layout.svelte:2]: Already resolved this

File: src/lib/components/Header.svelte (hunk 1 above)
Comment (JustinLung) [reply src/lib/components/Header.svelte]: github is just weird, in my code editor it is looking normal

File: src/routes/+layout.svelte (hunk 2 above)
Comment (boudewijnbout) [reply src/routes/+layout.svelte:2]: Nice!

File: src/styles/style.css (hunk 3 above)
Comment (JustinLung) [reply src/styles/style.css]: fixed this issue

**Comment by boudewijnbout** [comment]
LGTM!

**Pull request merged**